sklearn/gaussian_process/tests/test_kernels.py

```diff
@@ -323,3 +323,7 @@ def test_repr_kernels(kernel):
     # Smoke-test for repr in kernels.
 
     repr(kernel)
+
+
+def test_rational_quadratic_kernel():
+    assert_raises(AttributeError, RationalQuadratic, [1., 1.])
```

Comment:
I would use from `sklearn.utils.testing import assert_raise_message` to be more specific.